presto-native-execution/CMakeLists.txt

```diff
@@ -191,6 +199,8 @@ include_directories(${CMAKE_BINARY_DIR})
 # set this for backwards compatibility, will be overwritten in velox/
 set(VELOX_GTEST_INCUDE_DIR "velox/third_party/googletest/googletest/include")
 
+include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
```

Comment:
Curious why you had to add this. ?

Reply:
@kgpai 
It is in the description of the PR:

Add openssl@1 include directory BEFORE we add boost include directory, which happens to be "/Users//homebrew/include" and causes headers of openssl@3 being used instead of openssl@1.

Reply:
Yes I am curious about this - seems strange that boost should depend on openssl ..

Reply:
@kgpai 
It is not the case.
Here I just change order of include_directories calls, making sure openssl comes BEFORE "SYSTEM ${Boost_INCLUDE_DIRS}".

include_directories(SYSTEM ${Boost_INCLUDE_DIRS}) adds "/Users/homebrew/include", which is being before "/Users/spershin/homebrew/opt/openssl@1.1/include" would point to openssl@3.3's includes.

I don't know why include_directories(SYSTEM ${Boost_INCLUDE_DIRS}) adds generic "/Users/homebrew/include", but that's how it is and that caused issue with including openssl3 and linking with openssl1.